Wordle.Domain/GuessValidator.cs

```diff
@@ -0,0 +1,16 @@
+namespace Wordle.Domain;
+using System.Text.RegularExpressions;
+
+public class GuessValidator
+{
+    public bool IsFiveLetters(string guess)
+    {
+        return guess.Length == 5;
+    }
+
+    public bool IsValid(string guess)
+    {
+        // TODO: Guess should be validated against an actual dictionary
+        return IsFiveLetters(guess) && Regex.IsMatch(guess, @"^[a-zA-Z]+$");;
```

Comment:
I believe this regex, which determines if the `guess` consists of only "letters" should be in the `IsFiveLetters` function.

Reply:
Good point! Just pushed up these changes.

Reply:
Also,  you can use this in .net   guess.All(Char.IsLetter)  this takes a string then checks every char and the .net char class has a function to check if the char is a letter